frontend/test/metabase/scenarios/question/query-external.cy.spec.js

```diff
@@ -0,0 +1,35 @@
+import { restore } from "__support__/e2e/cypress";
+
+const supportedDatabases = [
```

Comment:
No Postgres?

Reply:
It wasn't there originally.
We will expand those tests now that they are together with the rest.